release/xgboost_tests/xgboost_tests.yaml

```diff
@@ -0,0 +1,20 @@
+- name: tune_small
+  owner:
+    mail: "kai@anyscale.com"
+    slack: "@tune-team"
+
+  cluster:
+    app_config: app_config.yaml
+    compute_template: tpl_cpu_small.yaml
+
+  run:
+    timeout: 600
+    script: workloads/tune_small.py
+    results: /tmp/tune_small.json
```

Comment:
Should this be passed in as an env var for the script, so it isn't hard-coded in two separate places?

Reply:
For artifacts we have the same problem - it's up to the test creator to make sure these names are synced. I'd keep it that way for now, as the `results` key can also be empty here (then no results will be fetched and just pass/no pass is reported)

Reply:
Ok, if the `results` key is set, that value will be used (and set as the TEST_OUTPUT_JSON env variable), and if it is not set, a random `/tmp/output.json` file will be used (and also passed as the TEST_OUTPUT_JSON)